migrations/20200915113207_van_survey_questions.js

```diff
@@ -138,7 +138,8 @@ exports.up = function(knex) {
           'api_key', json_build_object('__secret', v_api_key_ref),
           'van_system_id', queue_refresh_van_survey_questions.van_system_id,
           '__after', 'insert_van_survey_questions'
-        )
+        ),
+        'van-api'
```

Comment:
This could cause a refresh to be delayed while a 100k sync runs - let's have:
`van-api-refresh` and `van-api-sync`

Reply:
Oh ok, then we should use the graphile_worker priority feature to make resource refresh requests go before sync requests. Lower priority goes first